modules/installation-special-config-raid.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+//
+// * installing/install_config/installing_customizing.adoc
+
+[id="installation_special-config-raid_{context}"]
+= Configuring a RAID-enabled data volume
+
+You can enable software RAID partitioning to provide an external data volume.
+
+.Procedure
+
+. Create a machine config in the `<installation_directory>/openshift` directory that configures a data volume by using software RAID. In this example, it is called `raid1-alt-storage.yaml`:
```

Comment:
Change to '...machine **configuration**...'

Reply:
This is one of those situations where both "machine configuration" and "machine config" are used throughout the docs, and I'm not sure which is more important. It's "machine config" in this procedure in other versions, so I think I'll leave it to remain consistent 🙂